app/Http/Controllers/MainController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Post;
+use Illuminate\Http\Request;
+use Aws\S3\S3Client;
+use Illuminate\Support\Str;
+
+class MainController extends Controller
+{
+    protected S3Client $s3;
+
+    public function __construct(S3Client $s3) {
+        $this->s3 = $s3;
+    }
+
+    public function showMain()
+    {
+        $posts = Post::latest()->get();
+        return view('main', ['posts' => $posts]);
+    }
+
+    public function uploadPost(Request $request) {
+        $request->validate([
+            'image' => 'required|file|mimes:jpeg,jpg|max:1024', // max 1MB
```

Comment:
I tried to test in my local uploading an image with more than 1MB and it's not showing any error. That image post is not showing, but there is no error shown, mas @riandyrn 🤔

Reply:
The problem has been fixed in https://github.com/farulivan/jepret/pull/13/commits/003ac3540c6e1fb7b95600b2d36d278b4c0ab24c, @farulivan. Please kindly recheck.